### PR TITLE
[nrf noup] ci: use Standard Library in Jenkinsfile to link sub-projects together

### PR DESCRIPTION
**Pull request by thst-nordic**
Managing Docker version, Agent Labels, Github Status, manifest updates, downstream jobs from CI_LIB.

More info: https://projecttools.nordicsemi.no/confluence/display/NCS/Linking+Jenkins+Jobs+Together

Signed-off-by: Thomas Stilwell Thomas.Stilwell@nordicsemi.no

### CODE REVIEW

**Comment by thst-nordic** [comment]
These PR's all have very similar changes in order to link the CI plans together:     All are passing CI. 
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/979
https://github.com/NordicPlayground/fw-nrfconnect-zephyr/pull/174
https://github.com/NordicPlayground/fw-nrfconnect-mcuboot/pull/45
https://github.com/NordicPlayground/nrfxlib/pull/76

File: et HEAD~1 && git add -A && git commit -s -m [nrf noup] ci use Standard Library in Jenkinsfile -m Managing Docker version, Agent Labels, Github Status, manfest updates, downstream jobs from CI_LIB. && git push origin -f [hunk 1]
```diff
@@ -0,0 +1,6 @@
+* [33m31b475e[m[33m ([m[1;36mHEAD -> [m[1;32madd_ci_lib[m[33m, [m[1;31morigin/add_ci_lib[m[33m)[m [nrf noup] ci: use Standard Library in Jenkinsfile
```
Comment (carlescufi) [inline et HEAD~1 && git add -A && git commit -s -m [nrf noup] ci use Standard Library in Jenkinsfile -m Managing Docker version, Agent Labels, Github Status, manfest updates, downstream jobs from CI_LIB. && git push origin -f]: This file is surely a mistake?

File: Jenkinsfile [hunk 2]
```diff
@@ -56,27 +82,69 @@ pipeline {
...
+      echo "failure"
+      script { lib_Status.set("FAILURE", 'MCUBOOT', CI_STATE) }
       script{
         if (env.BRANCH_NAME == 'master' || env.BRANCH_NAME.startsWith("PR"))
         {
-            emailext(to: 'anpu',
-                body: "${currentBuild.currentResult}\nJob ${env.JOB_NAME}\t\t build ${env.BUILD_NUMBER}\r\nLink: ${env.BUILD_URL}",
-                subject: "[Jenkins][Build ${currentBuild.currentResult}: ${env.JOB_NAME}]",
-                mimeType: 'text/html',)
+            // emailext(to: 'anpu',
```
Comment (carlescufi) [inline Jenkinsfile]: what's with the commented out code?

**Comment by mbolivar** [comment]
> These PR's all have very similar changes in order to link the CI plans together: All are passing CI.
> [NordicPlayground/fw-nrfconnect-nrf#979](https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/979)

This doesn't update the west.yml file to set the mcuboot revision to `pull/45/head` for testing in CI. Is there another nrf pull request I'm missing here? That is needed just as it is in the zephyr case. In general, any changes to any repository other than nrf itself also need an update to nrf/west.yml.

Please see https://docs.zephyrproject.org/latest/guides/west/repo-tool.html for details.

File: et HEAD~1 && git add -A && git commit -s -m [nrf noup] ci use Standard Library in Jenkinsfile -m Managing Docker version, Agent Labels, Github Status, manfest updates, downstream jobs from CI_LIB. && git push origin -f (hunk 1 above)
Comment (thst-nordic) [reply et HEAD~1 && git add -A && git commit -s -m [nrf noup] ci use Standard Library in Jenkinsfile -m Managing Docker version, Agent Labels, Github Status, manfest updates, downstream jobs from CI_LIB. && git push origin -f]: removed

**Comment by thst-nordic** [comment]
pull/45/head was added to the nrf west.yml.    and CI passed. 
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/979/commits/22fd905ea9431cc08b14b9fc2e569f9fc4297362

File: Jenkinsfile (hunk 2 above)
Comment (thst-nordic) [reply Jenkinsfile]: it was causing spam.    Added back in.

**Pull request merged**